tests/subsys/tracing/test_ctf.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Script to handle trace data with uart backend.
+"""
+import os
+import sys
+import time
+import subprocess
+import pytest
+
+# Dumping the trace data in text format using babeltrace.
+@pytest.fixture(autouse=True)
+def uart_ctf_handle():
+    """ handle ctf data from uart backend """
+    #capture trace date for 5s.
+    time.sleep(5)
+    data_path = sys.argv[1]
+    os.environ['data_path'] = str(data_path)
+
+    if not os.path.exists(data_path + "/ctf"):
+        os.mkdir(data_path + "/ctf")
+
+    if os.path.exists("subsys/tracing/ctf/tsdl/metadata"):
+        os.system('cp subsys/tracing/ctf/tsdl/metadata ${data_path}/ctf/')
```

Comment:
It better to use "High-level file operations" to copy file.

from shutil import copyfile
copyfile(src, dst)

Reply:
Thanks Steven, Have changed  to copyfile.